### ISSUE TITLE
Core | Select issue: styling problem in iOS

### ISSUE THREAD

**Issue by misterjunio** [OnsenUI/OnsenUI#1864]
__Environment__

```
[Core]
    onsenui 2.1.0

[Framework]
    none

[Framework binding]
    none

[Additional libraries]
    none

[Platform]
    Mobile - iOS

[Browser]
    Mobile - iOS Safari
```

__Encountered problem__
Down arrow not shown near select box.

__How to reproduce__
Open the core example in any iOS Safari. No visual clue (down arrow) hints that there is a select box.

**Comment by asial-matagawa** [comment]
@misterjunio 
In iOS 8, it seems that `background-color: transparent;` removes the arrow.
I think this will not be fixed as long as we depend on the arrow which the native `select` provides...

**Comment by misterjunio** [comment]
@asial-matagawa are you saying we should add our own custom arrow?

**Comment by asial-matagawa** [comment]
@misterjunio
No, I think we should not make efforts to this problem too much since the number of iOS 8 users is getting lower.
I think we should prioritize fixing problems in iOS 9 and 10.
I will postpone fixing this issue for now...
If there is any problem, please feel free to ask me.

**Comment by asial-matagawa** [comment]
This problem is also reproduced in iOS 9 and 10.
The commit ID I used: https://github.com/OnsenUI/OnsenUI/tree/b5bc5c666d02fbafeac11581eb3b60473df9e9d1

**Comment by asial-matagawa** [comment]
This issue was superseded by #2017 and fixed in #2028.

Closing this.
